spec/requests/user_pages_spec.rb

```diff
@@ -1,416 +1,416 @@
+        page.evaluate_script('window.confirm = function() { return true; }')
+        within(:xpath, '//tr[contains(.//td/text(), "Andy")]') do
+          page.find(".fa-trash").click
+          current_path.should == users_path
+        end
+        page.should have_selector('.flash-success', text: 'User "Andy" was successfully deleted')
+        page.should_not have_selector('td', text: 'Andy')
+        page.should_not have_selector('td', text: 'andy@example.com')
+        page.should_not have_selector('td.delete', text: 'false')
+      end
+    end
+
+    describe "as a non-admin" do
+      before do
+        sign_in user
+        visit users_path
+      end
+
+      it "is inaccessible" do
+        current_path.should == root_path
+        page.should have_selector('.flash-alert', text: 'You are not authorized to access this page.')
+      end
+    end
+  end
+
+  describe "FEEDBACKS" do
+    let(:reviewer) { create(:user) }
+    let(:ac) { create(:associate_consultant) }
+    let(:review) { create(:review, associate_consultant: ac) }
+    let!(:feedback) { create(:feedback, review: review, user: reviewer) }
+
+    describe "as the reviewer" do
+      before do
+        sign_in reviewer
+      end
+
+      it "displays the feedback with a 'Resume Feedback' action if not submitted" do
+        visit feedbacks_user_path(reviewer)
+        page.should have_selector('#feedback-requests td', text: ac.user.name)
+        page.should have_selector('#feedback-requests td', text: review.review_type)
+        page.should have_selector('#feedback-requests td', text: review.feedback_deadline.to_s(:short_date))
+        page.should have_selector('#feedback-requests td', text: feedback.updated_at.to_s(:short_date))
+        page.should have_selector('#feedback-requests td .fa-pencil')
+        page.find(".fa-pencil").click
+        current_path.should == edit_review_feedback_path(review, feedback)
+      end
+
+
+      it "displays the feedback with a 'View Feedback' action if submitted" do
+        feedback.update_attribute(:submitted, true)
+        visit completed_feedback_user_path(reviewer)
+
+        page.should have_selector('#completeds td', text: ac.user.name)
+        page.should have_selector('#completeds td', text: review.review_type)
+        page.should have_selector('#completeds td', text: feedback.project_worked_on)
+        page.should have_selector('#completeds td', text: feedback.updated_at.to_date.to_s(:short_date))
+        page.should have_selector('#completeds td .fa-eye')
+        page.find(".fa-eye").click
+        current_path.should == review_feedback_path(review, feedback)
+      end
+    end
+
+    describe "as another user" do
+      before do
+        sign_in create(:user)
+        visit feedbacks_user_path(reviewer)
+      end
+
+      it "is inaccessible" do
+        current_path.should == root_path
+        page.should have_selector('.flash-alert', text: 'You are not authorized to access this page.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.